### DISCUSSION TITLE
【字幕增强】建议增加一个字幕自动加载

### DISCUSSION THREAD

**Discussion by jay763190097** [linuxdeepin/developer-center#2914] [Features Request & Ideas | 特性请求 & 头脑风暴]
![_ _20181123223255](https://user-images.githubusercontent.com/16317178/48948682-5e2a3480-ef70-11e8-9984-25c851cd431b.png)
下载了美剧电视剧和其字幕，每次都要手动加载字幕有点麻烦，建议可以选择文件夹，自动根据名称匹配对应的字幕加载。
功能类似以上这个

**Comment by ihipop** [comment]
直接可以设置把在线下载的字幕保存到电影所在文件夹就好了，字幕跟着电影走。看了代码现在的版本是不支持的，只能保存到电影设置目录

**Comment by ihipop** [comment]
@BLumia  考虑支持保存字幕到电影所在文件夹吗？ 这样字幕可以跟着电影走。射手以前都支持，而你们用的也是射手的API

**Comment by BLumia** [comment]
印象里是有同名加载的，可能是匹配规则不对？有时间再看看 - -

**Comment by ihipop** [comment]
@BLumia 是同名加载，但是你们网络匹配的字幕保存的目录在`~/.config` 电影拷走的时候，字幕不跟着走。

**Comment by jay763190097** [comment]
同名同目录没有匹配
